cmd/qat_plugin/README.md

```diff
@@ -89,7 +89,8 @@ $ kubectl describe node <node name> | grep intel.com/qat
 
      This command produces a Docker image named `ubuntu-demo-opencl`.
```

Comment:
We have a typo which I missed to fix. Can you please change this to:
This command produces a Docker image named crypto-perf

Reply:
Done!